.goreleaser.yml

```diff
@@ -74,7 +74,7 @@ dockers:
     goarch: amd64
 
     # Path to the Dockerfile (from the project root).
-    dockerfile: operator/Dockerfile
+    dockerfile: Dockerfile
```

Comment:
Note that goreleaser doesn't provide the necessary context or VERSION parameter for our Dockerfile, so this is probably not working.

Reply:
Yeah I also realized we're skipping the docker step during the dry run?

Reply:
Maybe we should remove the `dockers` section of the goreleaser if it isn't expected to work.